OpenOversight/migrations/versions/f01cb1bc6293_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
```

Comment:
It's always helpful to put a description here and in the filename, but not required :)